src/students/controller.js

```diff
@@ -1,51 +0,0 @@
-const pg = require("pg")
-
-const pool = new pg.Pool({
```

Comment:
How to differentiate between different environments such as dev/test/prod and how to differ between credentials as well.